webrender/src/renderer.rs

```diff
@@ -4026,7 +4026,7 @@ impl<B: hal::Backend> Renderer<B>
         //Note: this is a fake frame, only needed because texture deletion is require to happen inside a frame
         self.device.begin_frame();
         #[cfg(not(feature = "gleam"))]
-        self.device.wait_for_resources_and_reset();
+        self.device.wait_for_resources();
```

Comment:
I still think you should call the reset, because you won't use the command buffers anymore.